doc/index.rst

```diff
@@ -1,8 +1,45 @@
-Plutus
-======
+Plutus Programming and Reference Guide
+=============================================
+
+About Plutus
+--------------------
+
+Plutus consists of Plutus Core, the programming, scripting and assembly language; 
+and Plutus Tx, the compiler that compiles the Haskell source code into Plutus Core 
+to form the on-chain part of a contract application. All of this is used in combination 
+to write Plutus scripts that run on the Cardano blockchain. Compiled Plutus Core 
+scripts or programs are able to interact with the Cardano ledger through the 
+ledger interface. 
+
+The Plutus Core Repository
+----------------------------------
+
+The `Plutus Core Repository <https://github.com/input-output-hk/plutus>`_ contains 
```

Comment:
It contains a lot more than that! Maybe an appendix with a list of packages in the repo and their intros/usages/summary would be nice?

Reply:
Thanks for your feedback on this point. I think it deserves more attention, so I'll build on this further.

Reply:
At Ziyang's suggestion, I'm adding this: 

It also contains the Plutus Tx compiler and the libraries for writing Haskell code 
that can be compiled to Plutus Core.

Reply:
I've created a story in Jira to add a page listing the packages in the repo with descriptions, etc.